config/delegates.rb

```diff
@@ -230,6 +233,13 @@ def httpsource_resource_info(options = {})
         "uri" => EDEPOT_BASE_URL + uri,
         "headers" => {"Authorization" => ENV['HCP_AUTHORIZATION']}
       }
+    when 'wabo'
+      identifier = identifier.gsub('\\\\', '/')  # This is needed until a bug in the iiif-metadata-server is solved
```

Comment:
what bug?

Reply:
indien mogelijk link naar github issue of iets dergelijks.

Reply:
The metadata server gives urls back with double backslashes instead of single forward slashes. That needs to be solved, but since this project needs to run asap this was an easy fix on this side.